geopyspark/geotrellis/geotiff_rdd.py

```diff
@@ -1,37 +1,86 @@
-from geopyspark.geopycontext import GeoPyContext
+""""This module contains functions that create RDDs from files.
 
+There is only one function found within this module at this time, geotiff_rdd.
+"""
 
-class HadoopGeoTiffRDD(object):
-    def __init__(self, geopysc):
-        self.geopysc = geopysc
-        self._hadoop_wrapper = self.geopysc.hadoop_geotiff_rdd
 
-    def get_rdd(self, key_type, path, options=None):
-        key = self.geopysc.map_key_input(key_type, False)
+def geotiff_rdd(geopysc,
+                rdd_type,
+                uri,
+                options=None,
+                **kwargs):
 
-        if options is None:
-            result = self._hadoop_wrapper.getRDD(key, path, self.geopysc.sc)
-        else:
-            result = self._hadoop_wrapper.getRDD(key, path, options, self.geopysc.sc)
+    """Creates a RDD from GeoTiffs that are located on the local file system, HDFS, or S3.
+
+    Args:
+        geopysc (GeoPyContext): The GeoPyContext being used this session.
+        rdd_type (str): What the spatial type of the geotiffs are. This is
+            represented by the constants: SPATIAL and SPACETIME. Note: All of the
+            GeoTiffs must have the same saptial type.
+        uri (str): The path to a given file/directory.
+        options (dict, optional): A dictionary of different options that are used
```

Comment:
Available options should defiantly be included in the doc string, otherwise there is no way for people to know